src/protoc/Cargo.toml

```diff
@@ -12,9 +12,8 @@ path = "bin/protoc.rs"
 [dependencies]
 anyhow = "1.0.47"
 ore = { path = "../ore" }
-protobuf = "3.0.0-alpha.2"
-protobuf-codegen = "3.0.0-alpha.2"
-protobuf-codegen-pure = "3.0.0-alpha.2"
-protobuf-parse = "3.0.0-alpha.2"
+protobuf = { git = "https://github.com/MaterializeInc/rust-protobuf.git" }
```

Comment:
this commit is the only difference? https://github.com/MaterializeInc/rust-protobuf/commit/063964a8ab6fee98a9622a0c0a72a9765411025a

or can you summarize why we're forking?  It'll add maintenance cost for us so I want to make sure I understand

Reply:
Ah yeah, it's just that PR (https://github.com/stepancheg/rust-protobuf/pull/581). I dropped a note in deny.toml with a link to the PR.